test/integration/default/serverspec/cerner_tomcat_tester_spec.rb

```diff
@@ -169,10 +169,16 @@
   it { should be_owned_by 'my_user' }
   it { should be_grouped_into 'my_group' }
   it { should contain 'export TEST_VAR=TEST_VALUE' }
+  it { should contain 'CATALINA_PID="$CATALINA_BASE/bin/catalina.pid"' }
 end
 
 describe file('/etc/security/limits.d/my_user_limits.conf') do
   it { should be_file }
   it { should contain 'my_user - nofile 65536' }
   it { should contain 'my_user - nproc 4096' }
 end
+
+describe command('service tomcat_my_tomcat diagnostic') do
+  its(:stdout) { should match /Capturing JVM metrics of my_tomcat \(\d+\)\:/ }
```

Comment:
I had it exposed as a service command, as in the past, when we have had issues with a service, we generally are collecting a bundle of diagnostics. I added it to this service command so that it can be easily / consistently captured about the process on-demand (which is similar to what can invoked when it is having issues shutting the JVM down).